SearchImplem/GoogleAutocomplete.swift

```diff
@@ -73,14 +73,75 @@ class GoogleAutocomplete: UIViewController, UITableViewDataSource, UITableViewDe
                     }
                     for item in placePredictions {
                         let placeDescription = item["description"] as! String
-                        self.predictions.append(placeDescription)
+                        let googleID = item["place_id"] as! String
+                        let separatedStrings = placeDescription.components(separatedBy: ", ")
```

Comment:
I remember you already have separated values for these strings in the response. why not use those instead of separating them manually?

Reply:
Used the separated values